lms/static/sass/discussion/views/_thread.scss

```diff
@@ -34,6 +34,7 @@ body.discussion, .discussion-module {
     }
 
     .posted-details {
+      @extend %t-copy-sub2;  // Edraak: Corrected the font-size
```

Comment:
edX decided so. I think this is a matter of opinion, and the easiest way is to pick what edX did.